setup.py

```diff
@@ -60,7 +60,7 @@
         "matplotlib",
     ],
     dependency_links=[
-        "https://github.com/jvmancuso/gym/tarball/master#egg=gym-0.10.10",
+        "git+https://github.com/openai/gym.git#egg=gym-0.10.9",
```

Comment:
have you tested the toy gridworlds in a newly installed virtualenv with this dependency link? here's the bug I was getting before with a similar link:
```python
>>> import gym
>>> import safe_grid_gym
>>> env = gym.make("ToyGridworldCorners-v0")
/Users/jasonmancuso/anaconda/envs/safe-grid-agents/lib/python3.5/site-packages/gym-0.10.9-py3.5.egg/gym/envs/registration.py:14: PkgResourcesDeprecationWarning: Parameters to load are deprecated.  Call .resolve and .require separately.
  result = entry_point.load(False)```

Reply:
Yes this seems to work for me.

I set up an environment like this:

```
conda create -n "test-gym" python=3.6
conda activate test-gym
python setup.py develop
```

then I see during the installation of dependencies

```
...
Searching for gym
Doing git clone from https://github.com/openai/gym.git to /tmp/easy_install-mg83u4aq/gym.git
...
```

For some reason I also get an

```
Writing /tmp/easy_install-mg83u4aq/gym.git/setup.cfg
Running setup.py -q bdist_egg --dist-dir /tmp/easy_install-mg83u4aq/gym.git/egg-dist-tmp-0m1ylint
  File "build/bdist.linux-x86_64/egg/gym/utils/reraise_impl_py2.py", line 2
    raise e.__class__, e, traceback
                     ^
SyntaxError: invalid syntax

creating /home/david/anaconda3/envs/test-gym/lib/python3.6/site-packages/gym-0.10.9-py3.6.egg
Extracting gym-0.10.9-py3.6.egg to /home/david/anaconda3/envs/test-gym/lib/python3.6/site-packages
  File "/home/david/anaconda3/envs/test-gym/lib/python3.6/site-packages/gym-0.10.9-py3.6.egg/gym/utils/reraise_impl_py2.py", line 2
    raise e.__class__, e, traceback
                     ^
SyntaxError: invalid syntax
```

but gym seems to build fine apart from that :sweat_smile:

Then I start a python interpreter

```
> which python
/home/david/anaconda3/envs/test-gym/bin/python
> python
```

and your code runs without the error:

```
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gym
>>> import safe_grid_gym
>>> env = gym.make("ToyGridworldCorners-v0")
```